libr/core/cmd_print.c

```diff
@@ -437,7 +437,7 @@ R_API int r_core_process_input_pade(RCore *core, const char *input, char** hex,
 
 static void print_format_help(RCore *core) {
 	const char* help_msg[] = {
-	"pf:", "pf[.k[.f[=v]]|[ v]]|[n]|[0][ [sz] fmt] [a0 a1 ...]", "",
+	"pf:", "pf[.k[.f[=v]]|[v]]|[n]|[0|cnt][fmt] [a0 a1 ...]", "",
```

Comment:
maybe we should use a #define at the begining of the file instead of writing the same string 3 times